libs/jwst-rpc/src/context.rs

```diff
@@ -118,16 +97,10 @@ pub trait RpcContextImpl<'a> {
                     let mut updates = updates.lock().await;
                     if updates.len() > 0 {
                         debug!("save {} updates", updates.len());
-                        if let Some(update) = merge_updates(&id, &updates) {
-                            if let Err(e) = docs.write_update(id.clone(), &update).await {
+                        for update in updates.as_slice() {
```

Comment:
each client op (such as insert a char) will generate an update, so the server needs to provide a merging process of updates over a period of time to avoid writing a large number of updates into the database in a short time, we need to have similar process when we switch to s3 in the future, otherwise a large number of S3 Write OPS will be generated

Reply:
Now updates will contain `doc.guid` information in the beginning and they are not able to be merged.

We could decode the guids from update and make update vec into a `HashMap::<guid, updates>`.

Reply:
in yjs, it is assumed that users are editing the same document in most cases, so it will give priority to looking for items that need to be inserted near the last edited place when updating.

I think we can also use a similar strategy here, assuming that multiple users are synchronizing the same subdoc -- if the incoming updates's uuid are the same as the last update, they will be merged with the last update, and if they are different or exceeded the minimum update modify time, they will fall into the current strategy